fastplotlib/__init__.py

```diff
@@ -3,6 +3,8 @@
 from .layouts import Plot, GridPlot
 from .graphics import *
 from .graphics.selectors import *
+from .legends import *
```

Comment:
I think you might have checked this branch out from somewhere you were working on legends. I don't think legends should be in this PR.